src/sb_pointcloud_processing/src/ColourspaceConverter.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * Created by: Valerian Ratu
+ * Created on: 2018/01/13
+ * Description: A class which converts RGB pointclouds to the HSV colourspace.
+ */
+
+#include <ColourspaceConverter.h>
+
+ColourspaceConverter::ColourspaceConverter() {}
+
+void ColourspaceConverter::setInputCloud(PointCloud<PointXYZRGB>::Ptr input) {
+    cloud_ = input;
+}
+
+void ColourspaceConverter::filter(PointCloud<PointXYZHSV>& output) {
```

Comment:
filter usually implies the removal of unwanted items. Maybe a better function name would be pclXYZRGBAtoXYZHSV(). That way it's consistent with your other function. PointXYZRGBAtoXYZHSV().

Reply:
That's a bit verbose, meeting you halfway by changing it to `convert`